alacritty/src/config/mod.rs

```diff
@@ -390,6 +391,11 @@ pub fn installed_config(suffix: &str) -> Option<PathBuf> {
                     return Some(fallback);
                 }
             }
+
+            let fallback = PathBuf::from("/etc/alacritty").join(&file_name);
+            if fallback.exists() {
+                return Some(fallback);
+            }
```

Comment:
I think this should work `fallback.exists().then_some(fallback)` or `if else`, no need for `return` at the end of the function.